easysocial/easysocial/getalbums.php

```diff
@@ -80,7 +81,24 @@ public function get_albums()
 		// Creating object and calling relatvie method for data fetching.
 		$obj = new EasySocialModelAlbums;
 
-		$albums = $obj->getAlbums($uid, $type, $mydata);
+		if ($type == 'all')
+		{
+			// Getting all public photos
+			$options = array(
+				'pagination' => true,
+				'direction' => 'DESC',
+				'core' => false
+			);
+
+			$options['privacy'] = true;
+			$options['excludedisabled'] = true;
+			$options['withCovers'] = true;
```

Comment:
@Pratikshakhandagale  - either assign blank array to `$options` and then assign all key-value pair OR make a complete array of all key-value pair once and then assign it to `$options`.